client-v2/src/reducers/staleFrontsReducer.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import type { Action } from 'types/Action';
+
+type State = {
+  [string]: boolean
+};
+
+const staleFronts = (state: State = {}, action: Action): State => {
+  switch (action.type) {
+    case 'RECORD_STALE_FRONTS': {
+      const { payload } = action;
+      return { ...state, ...payload };
```

Comment:
I guess Flow might help us here, but (in tandem with the comment above) spreading directly into the state like this makes me feel irrationally uncomfortable.